openapi-cli/src/test/resources/ballerina-to-openapi/expected_gen/response/status_code_errors_01.yaml

```diff
@@ -0,0 +1,170 @@
+openapi: 3.0.1
+info:
+  title: PayloadV
+  version: 0.0.0
+servers:
+  - url: "{server}:{port}/payloadV"
+    variables:
+      server:
+        default: http://localhost
+      port:
+        default: "9090"
+paths:
+  /statusCodeError:
+    get:
+      operationId: getStatuscodeerror
+      parameters:
+        - name: statusCode
+          in: query
+          required: true
+          schema:
+            type: integer
+            format: int64
+      responses:
+        "451":
+          description: UnavailableDueToLegalReasons
+        "431":
+          description: RequestHeaderFieldsTooLarge
+        default:
+          description: Default response
+        "410":
+          description: Gone
+        "411":
+          description: LengthRequired
+        "510":
+          description: NotExtended
+        "412":
+          description: PreconditionFailed
+        "511":
+          description: NetworkAuthenticationRequired
+        "413":
+          description: PayloadTooLarge
+        "415":
+          description: UnsupportedMediaType
+        "416":
+          description: RangeNotSatisfiable
+        "417":
+          description: ExpectationFailed
+        "421":
+          description: MisdirectedRequest
+        "400":
+          description: BadRequest
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorPayload'
+        "422":
+          description: UnprocessableEntity
+        "401":
+          description: Unauthorized
+        "500":
+          description: InternalServerError
+          content:
+            application/json:
+              schema: {}
```

Comment:
Is there a reason for having this `InternalServerError` with an empty schema instead of the `ErrorPayload`?

Reply:
Here the `InternalServerError` corresponds to the `httpscerr:InternalServerErrorError` type which is returned from the resource. For this status code error, the body is optional and default type is `anydata`. So ideally there should not be a content since the body is optional. Will fix that

Reply:
Fixed via [4ae8d86](https://github.com/ballerina-platform/openapi-tools/pull/1604/commits/4ae8d8652f0245bcf25ebb15db8f268844a02fa8)